week-8/Homework/mandatory/2-fetch-exercise/exercise.js

```diff
@@ -17,10 +16,21 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
-    .then(function(response) {
-        return response.text();
-    })
-    .then(function(greeting) {
-        // Write the code to display the greeting text here
-    });
\ No newline at end of file
+fetch("https://codeyourfuture.herokuapp.com/api/greetings")
+  .then(function (response) {
+    return response.text();
+  })
+  .then(function (greeting) {
+    greetingWords(greeting);
+    console.log(greeting);
+
+    // Write the code to display the greeting text here
+  });
+
+let getHtmlDoc = document.getElementById("greeting-text");
+
+function greetingWords(greet) {
+  let createH2 = document.createElement("H2");
+  createH2.textContent = greet;
+  getHtmlDoc.appendChild(createH2);
+}
```

Comment:
You got wrong here, the random greeting should be displayed in the box not to display it in a h2. It needs to be fixed.